### ISSUE TITLE
trouble with the start.txt file

### ISSUE THREAD

**Issue by wujastyk** [chchch/upama#59]
Could you have a look at 
* [https://saktumiva.org/wiki/wujastyk/susrutasamhita/start](https://saktumiva.org/wiki/wujastyk/susrutasamhita/start)

It displays fine on my local system, but not on Saktumiva.  File attached, as the MD source file and the exported html.
[start.md](https://github.com/chchch/upama/files/12063854/start.md)
[start.txt](https://github.com/chchch/upama/files/12063856/start.txt)

**Comment by chchch** [comment]
For non-TEI pages, the system expects markdown, so if you just use the markdown instead of HTML it should look right-ish.

**Comment by wujastyk** [comment]
I don't think that can be right.  I've tried using markdown and it doesn't work (with either a .md or a .txt extension).

I have always written this file in markdown, exported it to HTML and then ftp'ed it to Saktumiva.  Always worked just fine.  Now, the file is there, as html, but it's not rendering as HTML, ut as plain text.

![image](https://github.com/chchch/upama/assets/762246/4191519c-7048-4654-afa6-7376254c80ba)

**Comment by chchch** [comment]
I tried saving your `start.md` file as `start.txt` and it worked. But your tables seem to be really wide because the links are in a different "Markdown" flavour... I guess DokuWiki expects MediaWiki style links or something (e.g., `[[https://saktumiva.org/wiki/wujastyk/susrutasamhita/01-su.su-32-end/provisional-edition_sutrasthana-32-end?upama_ver=gv6fam6pom&upama_script=devanagari&upama_scroll=SS.1.41.1|41]]`)

**Comment by wujastyk** [comment]
I'm flummoxed by all this.  For over a year, I think, I've been working fine with this procedure:

* start.md exported by pandoc to start.html
* start.html renamed start.txt
* start.txt uploaded to Saktumiva

Clumsy, but it worked just fine.  Until a week or two ago. I've never used .md files at Saktumiva.  I'm sure you told me to use the above procedure, long ago (i.e., html content, file named .txt).

But now Saktumiva isn't rendering the html any more, just displaying as plain text.  Something has definitely changed in Saktumiva's behaviour. 

Since my .md files ("Github Flavoured Markdown" from Typora) aren't rendering properly, would it be possible to just have start.txt auto-redirect to an external website?  Then I can do what I want on WordPress or Github.  I tried the obvious `<meta http-equiv="refresh" content="0;URL='http://thetudors.example.com/'" /> ` and it didn't work (because Saktumiva's not seeing it as HTML).  Is there some other workaround?

Incidentally, this is what my .md file really looks like:

![image](https://github.com/chchch/upama/assets/762246/461059a4-0772-4590-8fc8-a2bbe779d671)

This same rendering can also be seen [at Github](https://github.com/wujastyk/sushrutaproject/blob/main/start.md).

**Comment by wujastyk** [comment]
Hooray!  Thank you, Charles.  Everything is fine again.
